js/src/services/team-api.ts

```diff
@@ -78,7 +78,7 @@ export const changeStatus = ({
   api
     .post(`/teams/${teamId}/${status}`, {})
     .then<Success<null>>(() => ({ success: true, data: null }))
-    .catch<FormFailure>((resp) => ({
+    .catch<Failure>((resp) => ({
       success: false,
-      errors: resp?.response?.data?.errors || [],
+      reason: "Something went wrong",
```

Comment:
Reason should be like a constant or variable name, e.g. 

```ts
reason: "something_went_wrong"
```

Reply:
Done